src/index.js

```diff
@@ -48,7 +48,7 @@ import isHSL from './lib/isHSL';
 
 import isISRC from './lib/isISRC';
 
-import isIBAN from './lib/isIBAN';
+import isIBAN, { locales as isIBANLocales } from './lib/isIBAN';
```

Comment:
May be calling them `isIBANLocales` could be a little misleading, might think it's a validator/function. Perhaps just drop the `is`?

Reply:
@profnandaa I see what you're saying, but I think it's more inline with the rest of the code to call it `isIBANLocales`